modules/learn/pages/clusters-and-availability/rebalance.adoc

```diff
@@ -164,6 +164,32 @@ If more index replicas exist than can be handled by the number of existing nodes
 During rebalance, no index node is removed until index-building has completed on alternative nodes.
 This ensures uninterrupted access to indexes.
 
+==== Smart Batching
+
+When rebalance occurs, indexes and relocated and rebuilt in _batches_, so as to place a limit on the amount of work that is attempted concurrently -- the default batch size being `3`.
```

Comment:
"indexes and relocated" --> "indexes are relocated"

Reply:
Thanks, have changed.